tests/test_datetime_encoder.py

```diff
@@ -18,14 +22,18 @@ def test_cyclical_feature(self):
         encoder.fit(self.df)
         np_array = encoder.transform(self.df)
         assert np_array.shape == (len(self.df), 16)
+
     def test_datetime64_expand_encoder(self):
         encoder = Datetime64ExpandEncoder()
         encoder.fit(self.df)
         np_array = encoder.transform(self.df)
         assert (np_array[0] == np.array([2009, 12, 1, 23, 19, 1, 335, 4])).all()
+
     def test_dummy_encoder(self):
         encoder = DummyEncoder()
         encoder.fit(self.df)
         np_array = encoder.transform(self.df)
-        print(np_array)
-        assert np_array.shape == (len(self.df), 4 * len(self.df) + 4)
+        if int(pd.__version__[0]) < 2:
+            assert np_array.shape == (len(self.df), 4 * len(self.df) + 4)
+        else:
+            assert np_array.shape == (len(self.df), 4 * len(self.df))
```

Comment:
Why is there a difference here? Is it worth documenting the difference here?

Reply:
Same question as Aecio.